tests/config_test.py

```diff
@@ -699,6 +699,17 @@ def testConfigurableMethod(self):
     instance, _ = configurable2()  # pylint: disable=no-value-for-parameter
     self.assertEqual(instance.implement_me(), 'bananaphone')
 
+  def testConfigurableWithFunctioncall(self):
+    config_str = """   
+          M = 'macro'       
+          ConfigurableClass.kwarg1 = @configurable2(non_kwarg='statler')
+          ConfigurableClass.kwarg2 = @configurable2(non_kwarg=%M, kwarg1='waldorf')
```

Comment:
Can you also add tests like `@configurable2(non_kwarg=@configurable2(non_kwarg='sth'))`

Reply:
added